tools/protease/protease.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="protease" name="Protease Model Driver" version="@VERSION@">
```

Comment:
The name should be changed and the ID as well. Currently it's not clear for the user what this tool is doing.